test/Parse/raw_string.swift

```diff
@@ -68,6 +68,46 @@ _ = ##"""
       """##
 // CHECK: "a raw string with \"\"\" in it"
 
+// ===---------- False Multiline Delimiters --------===
+
+/// Source code contains zero-width character in this format: `#"[U+200B]"[U+200B]"#`
+/// The check contains zero-width character in this format: `"[U+200B]\"[U+200B]"`
+/// Use this test when implementating `diagnoseZeroWidthMatchAndAdvance`.
```

Comment:
I might be a little more specific, perhaps:

```suggestion
/// If this check fails after you implement `diagnoseZeroWidthMatchAndAdvance`,
/// then you may need to tweak how to test for single-line string literals that
/// resemble a multiline delimiter in `advanceIfMultilineDelimiter` so that it
/// passes again.
```